plugin.video.lbry/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.lbry" name="LBRY" version="0.1.1" provider-name="accumulator">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.routing" version="0.2.3"/>
+        <import addon="script.module.requests" version="2.22.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="lbry.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Plugin for LBRY</summary>
+        <description lang="en_GB">LBRY is a secure, open, and community-run digital marketplace</description>
+        <language>en_GB</language>
```

Comment:
please remove the language tag, this applies to content only. If you want to limit this to english, please use `en`